static/app/utils/replays/hooks/useReplayOnboarding.tsx

```diff
@@ -42,10 +42,10 @@ export function useReplayOnboardingSidebarPanel() {
   const hasSentOneReplay = useHaveSelectedProjectsSentAnyReplayEvents();
 
   useEffect(() => {
-    if (hasSentOneReplay && location.hash === '#replay-sidequest') {
+    if (location.hash === '#replay-sidequest') {
       SidebarPanelStore.activatePanel(SidebarPanelKey.ReplaysOnboarding);
     }
-  }, [hasSentOneReplay, location.hash]);
+  }, [location.hash]);
 
   const activateSidebar = useCallback((event: {preventDefault: () => void}) => {
```

Comment:
Should it not call this one?

Reply:
Actually does this function still need to call `activatePanel`, since now the use effect will trigger that

Reply:
what could be calling that one? 

The `useEffect` above doesn't have an `event` param to pass along.